addons/website/static/src/core/colibri.js

```diff
@@ -178,7 +178,23 @@ export class Colibri {
                 const attr = directive.slice(6);
                 const initialValues = new Map();
                 for (const node of nodes) {
-                    const attrValue = node.getAttribute(attr);
+                    let attrValue;
+                    switch (attr) {
+                        case "class":
+                            attrValue = [...node.classList];
+                            break;
+                        case "style":
+                            attrValue = {};
+                            for (const property of Object.keys(value())) {
```

Comment:
`value()` does not seem right. Don't you mean `value` ?

Reply:
no, it's correct. in the case of `t-att-style`, the value is a function